### PR TITLE
adding the example game to the server

### PR DESCRIPTION
**Pull request by arielofer**
adding a file with a list of scenes and using the gamebook-maker library to host the game on the server

### CODE REVIEW

**Comment by arielofer** [comment]
the server and client code crashes when the client socket closes.
need to build handlers

File: game_manager_example.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+                Option('Go right', user_inputs=['r', 'right'],
+                       next_scenes=[NextScene(next_scene2)])
+            ]
+        )
+
+scenes_list = [intro_scene, next_scene1, next_scene2, dead_scene, lucky_scene,
+               flee_scene]
+
+
+"""def main():
```
Comment (agamm) [inline game_manager_example.py]: Why is this here?

File: server.py [hunk 2]
```diff
@@ -19,16 +19,17 @@
...
         input_sender = NetworkTerminalInput(client)
         with client:
-            # gm = GameManager(scenes_list, output_sender, input_sender)
+            gm = GameManager(scenes_list, output_sender, input_sender)
             server_output.output(f"connection made from {address}")
             # client.settimeout(15)
-            while True:
-                # gm.start(intro_scene)
-                data = input_sender.input("enter a word: ")
+            # while True:
```
Comment (agamm) [inline server.py]: Don't keep comments you aren't using.

File: server.py [hunk 3]
```diff
@@ -19,16 +19,17 @@
...
+            gm.start(intro_scene)
```
Comment (agamm) [inline server.py:24]: Wouldn't it start the same scene each time?

**Review by agamm — CHANGES_REQUESTED** [review]

File: server.py (hunk 3 above)
Comment (arielofer) [reply server.py:24]: no, because `gamemanager.start()` uses the `.get_next_scene()` method
   
` def start(self, current_scene):`
        `self.current_scene = current_scene`
       `while True:`
            `next_scene_name = self.run_scene()`
           `self.current_scene = self.get_next_scene(next_scene_name)`

Comment (agamm) [reply server.py:24]: Ok, so why do you have the while True before the `GameManager` ? are you planning of serving multiple clients in parallel?

**Review by agamm — APPROVED** [review]

**Comment by arielofer** [comment]
i wanted to, but i have to send the book project today. i don't have the time to add it
